public/api/wekan.html

```diff
@@ -1458,12 +1458,12 @@
   opacity: 0.5;
 }
 </style>
-    
+
```

Comment:
Changes to this file should be removed. Wekan API docs are generated with script, they should be not modified manually.